cdap-cli/src/main/java/co/cask/cdap/cli/command/CreateDatasetInstanceCommand.java

```diff
@@ -52,21 +53,32 @@ public void perform(Arguments arguments, PrintStream output) throws Exception {
     String datasetName = arguments.get(ArgumentName.NEW_DATASET.toString());
     String datasetPropertiesString = arguments.getOptional(ArgumentName.DATASET_PROPERTIES.toString(), "");
     String datasetDescription = arguments.getOptional(ArgumentName.DATASET_DESCRIPTON.toString(), null);
+    String datasetOwner = arguments.getOptional(ArgumentName.OWNER_PRINCIPAL.toString(), null);
+    KerberosPrincipalId ownerPrincipal = datasetOwner != null ? new KerberosPrincipalId(datasetOwner) : null;
+
     Map<String, String> datasetProperties = ArgumentParser.parseMap(datasetPropertiesString);
     DatasetInstanceConfiguration datasetConfig =
-      new DatasetInstanceConfiguration(datasetType, datasetProperties, datasetDescription);
+      new DatasetInstanceConfiguration(datasetType, datasetProperties, datasetDescription, ownerPrincipal);
 
     datasetClient.create(cliConfig.getCurrentNamespace().dataset(datasetName), datasetConfig);
-    output.printf("Successfully created dataset named '%s' with type '%s' and properties '%s'",
-                  datasetName, datasetType, GSON.toJson(datasetProperties));
+    StringBuilder builder = new StringBuilder(String.format("Successfully created dataset named '%s' with type " +
+                                                              "'%s', properties '%s'", datasetName, datasetType,
+                                                            GSON.toJson(datasetProperties)));
+    if (datasetDescription != null) {
+      builder.append(String.format(" , description '%s'", datasetDescription));
+    }
+    if (datasetOwner != null) {
+      builder.append(String.format(" , owner principal '%s'", datasetOwner));
+    }
+    output.printf(builder.toString());
     output.println();
   }
 
   @Override
   public String getPattern() {
-    return String.format("create dataset instance <%s> <%s> [<%s>] [<%s>]",
+    return String.format("create dataset instance <%s> <%s> [<%s>] [<%s>] [<%s>]",
```

Comment:
How will having three optional parameters work in practice? Should this not be:
```
"create dataset instance <%s> <%s> [<%s> [<%s> [<%s>]]]"
```
where you put on either one, two, or all three optional parameters?

Or as you can see in http://builds.cask.co/artifact/CDAP-DQB248/shared/build-1/Docs-HTML/4.1.0-SNAPSHOT/en/reference-manual/cli-api.html#egress you need to preface the parameters with an identifier.
If the description is not included, how does the CLI know that you intend for the second parameter to be an `owner-principal`?

What is the form of the `owner-prinicpal`, and can we add some docs to the CLI to help users?